product_company_default/readme/DESCRIPTION.md

```diff
@@ -0,0 +1,2 @@
+This module populates the company field with user's company as the
```

Comment:
This is not what the module does. Is it normal ?

Moreover, using the config parameter with a company that is not enabled on user that creates the product can leads to UX problems.

Reply:
I think the current description is not accurate. Should be "the current active one" instead of "user's company."

> Moreover, using the config parameter with a company that is not enabled on user that creates the product can leads to UX problems.

@rousseldenis I'm not getting this comment. The config parameter doesn't depend on the company.